core/storage/user/gae_models.py

```diff
@@ -183,7 +190,12 @@ def get_export_policy(cls):
             'first_contribution_msec':
                 base_models.EXPORT_POLICY.EXPORTED,
             # Pin is not exported since this is an auth mechanism.
-            'pin': base_models.EXPORT_POLICY.NOT_APPLICABLE
+            'pin': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            # TODO(#12755): Currently, "roles" and "banned" fields are not in
```

Comment:
You can drop this TODO, I think. It won't result in any changes here.

Reply:
Ah -- I see. I agree, but I think the issue for me was that this TODO didn't seem to have a clear action. Could you write it (and TODOs in future) as: "TODO(#...): Change these to XXX." and then give the explanation? That way, it is clear what the concrete action is.

Reply:
Done!

I was following this pattern: `<current unexpected behavior with reason> <expected behavior> <action to change from unexpected to expected behavior>`

But I think for todos having: `<todo-action> <reason>` should be better as after `TODO` we expected to see the expected actions/changes needs to be done.

Thanks for the suggestion!